web/e2e/page-objects/dashboard/dashboard.ts

```diff
@@ -43,6 +43,28 @@ export class DashboardPage extends AppPage {
     await browser.sleep(2000);
   }
 
+  async clearStagingDatabase() {
+    await browser.wait(EC.visibilityOf(this.clearStagingButton()));
+    await this.clearStagingButton().click();
+    await browser.sleep(500);
```

Comment:
if you need to sleep here, might as well to put it to 1000 rather than 500. 500 is 0.5 sec

Reply:
I noticed sometimes protractor works more stable if I add a short sleep timeout like after clicking on a button. Generally, we need to remove all timeout or make them as short as possible. In this particular case, 0.5 sec. timeout works well.